.travis.yml

```diff
@@ -1,6 +1,13 @@
 language: node_js
 before_install:
   - npm install -g npm@latest
+install:
+  - npm install --global electron
+  - npm install
+before_script:
+  - export DISPLAY=:99.0; sh -e /etc/init.d/xvfb start
```

Comment:
Would it make sense for this to be using the same `xvfb-run` as https://github.com/resin-io/etcher/blob/master/.travis.yml#L60 ?